server-spi/src/main/java/org/keycloak/models/ClientScopeModel.java

```diff
@@ -67,6 +67,9 @@
     String CONSENT_SCREEN_TEXT = "consent.screen.text";
     String GUI_ORDER = "gui.order";
     String INCLUDE_IN_TOKEN_SCOPE = "include.in.token.scope";
+    String LOGO_URI ="logoUri";
```

Comment:
I am not sure why these constants are defined on the ClientScopeModel? Existing constants on ClientScopeModel are the configuration options of client scope. However these constants logoUri, policyUri, tosUri are not config options of clientScopeModel. It seems the better place for define them might be ClientModel class or "org.keycloak.models.Constants"

Reply:
You are right. I move this constants to ClientModel.